README.md

```diff
@@ -1,2 +1,74 @@
 # k3s-local-registry
-Coming soon!
+When using Tilt with a [k3s](https://k3s.io/) cluster, we recommend using a local registry for faster image pushing and pulling.
```

Comment:
> for faster image pushing and pulling...

I joinked some language from the `kind-local` repo and didn't quite know how to say: we're not recommending this approach as much for perf characteristics (afaict `k3d import-images` doesn't actually suck) as because we don't have built-in support for calling `k3d import-images` in place of `docker push`). Idk if we need to provide that detail here tho...